### ISSUE TITLE
Not working on 6.8.0-39 Kernel with Linux Mint 22

### ISSUE THREAD

**Issue by Flofler** [AndiWeiss/alx-wol#16]
Unfortunately this tool doesn't seem to work anymore after upgrading from Linux Mint 21.3 to Linux Mint 22. Kernel was upgraded from 5.15.0.117 to  6.8.0-39.

I completely reinstalled my system with Linux Mint 22 and applied your tool but my network driver still doesn't support WOL.
Do you have any idea what the issue could be?

**Comment by AndiWeiss** [comment]
Hi there,
well, the point is: my mechanism tries to get the kernel version.
To be able to do so the scripting extracts the kernel version strings out of the kernel.
And up to now Mint linux isn't foreseen in the scripting.
I started with Ubuntu, went to Debian and then Proxmox. Last distro I added is Arch.
I'll take a look for mint as next step, shouldn't be a big thing as it bases on Ubuntu.
But it would be helpful to get the output of the install.sh run ...
